ANMN/burst_averaged_product/burst_average_product.py

```diff
@@ -0,0 +1,388 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Burst Average Product creation for WQM and CTD FV01 files
+./burst_average_product.py input_netcdf.nc /ouput_dir
```

Comment:
You could keep the script name short and just call it "burst_average.py".  Otherwise it should be "burst_average**d**_product.py".